### ISSUE TITLE
AutoClose issue when opening dropdown using isOpen property

### ISSUE THREAD

**Issue by tanishkab** [valor-software/ng2-bootstrap#6422]
Hi, I am kind of facing somewhat same issue, but here I am using isOpen property of dropdown in my ts file to open it,  but I want the dropdown to autoClose and it is not happening.

 `<div class="btn-group" dropdown [insideClick]="true" #newdropdown="bs-dropdown" [autoClose]="true" container="body">`
        `               <div *dropdownMenu class="dropdown-menu" role="menu">`
                `<multiEntity-table> </multiEntity-table>`
        ` </div>`
   `</div>`
   
   I am using #newdropdown template ref to use isOpen property in my .ts file.

_Originally posted by @tanishkab in https://github.com/valor-software/ngx-bootstrap/issues/5222#issuecomment-1020895728_

**Comment by SvetlanaMuravlova** [comment]
@tanishkab can you add more details regarding your issue, what version of library and angular do you use?

**Comment by tanishkab** [comment]
Angular version : "~11.1.0" ,
ngx-bootstrap version: "^6.1.0"

**Comment by SvetlanaMuravlova** [comment]
try to use the next version 6.2.0, and i can see the property autoclose is workable in such version
https://valor-software.com/ngx-bootstrap/old/6.2.0/#/dropdowns

**Comment by tanishkab** [comment]
The problem is still there, even after upgrading the library

**Comment by SvetlanaMuravlova** [comment]
@tanishkab, so i think it will be helpful if you will create a small example to reproduce such bug, cause we can't reproduce it
Also i can see it works in our documentation.

**Comment by tanishkab** [comment]
Stackblitz representation: https://stackblitz.com/edit/angular-ngx-dropdown-msaj1m?file=src/app/app.component.ts

**Comment by SvetlanaMuravlova** [comment]
@tanishkab thank you for your message, i can confirm your issue now

**Comment by moddi3** [comment]
I also have this problem(

**Comment by liuyankit** [comment]
I'm encountering the same issue, anyone has a workaround?

**Comment by tanishkab** [comment]
nope, if you can add dropdownToggle then it works fine otherwise not
